tests/test_lcc.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#%%
+from earth2grid.lcc import HRRR_CONUS_GRID
+import numpy as np
+import torch
+from pytest import approx
```

Comment:
nit: For 3rd party packages, I roughly try to follow [Google's style guide for imports](https://google.github.io/styleguide/pyguide.html#22-imports) which forbids importing members from modules.


```suggestion
import pytest
```

and then `pytest.approx` below.